docs/learning-the-basics.md

```diff
@@ -102,25 +102,25 @@ The panel, dock, and workspaces appear only on the primary display. Any other at
 
 ## Installing Apps {#installing-apps}
 
-elementary OS comes bundled with **AppCenter**, an app store for free apps. Installing a new app from AppCenter is easy:
+elementary OS comes bundled with **AppCenter**, an app store for apps free of charge. Installing a new app from AppCenter is easy:
 
 1. Open the **AppCenter** app.
 2. Search in the top-right, or browse.
 3. Click **Install** next to the app you want to install.
 
 You may be asked for your password prior to installing an app.
 
-_Note: Some software may not be available from AppCenter. While we don't recommend downloading software from the general Internet, apps that are compatible with Ubuntu 16.04 LTS should work on elementary OS Loki._
+_Note: Some software may not be available from AppCenter. While downloading software from the general Internet is not recommended, apps compatible with Ubuntu 16.04 LTS should work on elementary OS Loki._
```

Comment:
I don't like the more passive voice here.

Reply:
It begs the question of who "we" are.